Appsilon/rhino
# Box universal import linter

### Changes
- Closes #532 
- Adds a `box_universal_import_linter()`
- Adds the linter to the `.lintr` template
- Adds documentation

### How to test
1. Unit tests for `box_universal_import_linter()` exist
2. Locally install `rhino`, version should be `1.7.0.9001`, initialize a new rhino application then proceed to declare box imports.
3. `rhino::lint_r()` should find and point to all `package[...]` and `path/to/file[...]` imports.
4. Or, with an existing rhino app, add `box_universal_import_linter = rhino::box_universal_import_linter()` to `.lintr`.
5. `rhino::lint_r()` should find and point to all `package[...]` and `path/to/file[...]` imports.
